server/server/src/sync/sync_api_v5.rs

```diff
@@ -107,13 +107,13 @@ impl SyncApiV5 {
     }
 
     // Get batch of records from remote sync queue.
-    pub async fn get_queued_records(&self) -> Result<RemoteSyncBatchV5, SyncConnectionError> {
-        // Arbitrary batch size.
-        const BATCH_SIZE: u32 = 500;
-
+    pub async fn get_queued_records(
+        &self,
+        batch_size: u32,
+    ) -> Result<RemoteSyncBatchV5, SyncConnectionError> {
         let url = self.server_url.join("/sync/v5/queued_records")?;
 
-        let query = [("limit", &BATCH_SIZE.to_string())];
+        let query = [("limit", &batch_size.to_string())];
```

Comment:
since we want to export full initialisation, batch_size needs to be specified when calling the pulls